frontend/src/app/services/validate.ts

```diff
@@ -38,5 +36,7 @@ export function validateGame(data: GameData, breakImmediately = true): Game | un
 
   if (!res) console.error(`Validation failed, new data: ${JSON.stringify(game.simulation.modelStates, null, 2)}`);
 
+  // Set the real scenario for the future gameplay mitigations
+  game.scenario = new Scenario(data.scenario.dates, data.scenario.gameplayMitigationHistory);
```

Comment:
I kind of dislike this duality. And also the duality of gameplay and ramp-up mitigations. Could not we join these pairs somehow? To be discussed offline...